### PR TITLE
FIX: broken tag

### PR DESCRIPTION
**Pull request by pratapvardhan**
A tag was not closed properly, that's messing up the render. This should probably fix it.

### CODE REVIEW

File: _notebooks/2020-03-17-covid19-compare-country-death-trajectories.ipynb [hunk 1]
```diff
@@ -290,7 +290,7 @@
    ],
    "source": [
     "#hide_input\n",
-    "HTML(f'<small class=\"float-right\">Last Updated on {pd.to_datetime(LAST_DATE).strftime(\"%B, %d %Y\")}<small>')"
+    "HTML(f'<small class=\"float-right\">Last Updated on {pd.to_datetime(LAST_DATE).strftime(\"%B, %d %Y\")}</small>')"
```
Comment (pratapvardhan) [inline _notebooks/2020-03-17-covid19-compare-country-death-trajectories.ipynb]: This one.

File: _notebooks/2020-03-17-covid19-compare-country-trajectories.ipynb [hunk 2]
```diff
@@ -289,7 +289,7 @@
    ],
    "source": [
     "#hide_input\n",
-    "HTML(f'<small class=\"float-right\">Last Updated on {pd.to_datetime(LAST_DATE).strftime(\"%B, %d %Y\")}<small>')"
+    "HTML(f'<small class=\"float-right\">Last Updated on {pd.to_datetime(LAST_DATE).strftime(\"%B, %d %Y\")}</small>')"
```
Comment (pratapvardhan) [inline _notebooks/2020-03-17-covid19-compare-country-trajectories.ipynb]: and this one.

**Comment by pratapvardhan** [comment]
@hamelsmu -- made a tiny fix for `</small>` closure tag.

**Comment by hamelsmu** [comment]
@pratapvardhan Thanks for staying on top of this!  I think we both noticed this issue at the same time, and I just deployed the same fix 🙇 

Appreciate it!

**Pull request closed without merging**